sheer/query.py

```diff
@@ -130,9 +130,10 @@ def __iter__(self):
             for hit in self.result_dict['hits']['hits']:
                 yield QueryHit(hit)
 
-    def facets(self, fieldname):
-        if "facets" in self.result_dict and fieldname in self.result_dict["facets"]:
-            return self.result_dict['facets'][fieldname]["terms"]
+    def aggregations(self, fieldname):
```

Comment:
:+1:  

While I was going through the Sheer source and Elasticsearch documentation, I saw that facets are deprecated in favor of aggregations.